packages/@aws-cdk/aws-elasticloadbalancingv2/lib/alb/application-load-balancer.ts

```diff
@@ -65,6 +66,7 @@ export class ApplicationLoadBalancer extends BaseLoadBalancer implements IApplic
       ipAddressType: props.ipAddressType,
     });
 
+    this.ipAddressType = props.ipAddressType || IpAddressType.IPV4;
```

Comment:
```suggestion
    this.ipAddressType = props.ipAddressType ?? IpAddressType.IPV4;
```

preferred to use nullish coalescing now that we're on TypeScript 3.7+